File: FertiWeb/src/main/java/dominio/Analisis.java
package dominio;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Analisis {

	private Long codigoAnalisis;
	private Date fechaAnalisis;
	private Long codigoParcela;

}

File: FertiFront/src/app/clases_dominio/analisis.ts
import { Parcela } from './parcela';
import { ElementoXAnalisis } from './elemento-x-analisis';

export class Analisis {

    public codigoAnalisis: number;
    public parcela: Parcela;
    public fechaAnalisis: Date;
    public elementos: ElementoXAnalisis[];
}
